Source/WebCore/rendering/svg/SVGTextLayoutEngine.cpp

```diff
@@ -266,17 +268,17 @@ void SVGTextLayoutEngine::finalizeTransformMatrices(Vector<InlineIterator::SVGTe
     if (textBoxes.isEmpty())
         return;
 
-    for (auto textBox : textBoxes) {
-        auto textBoxTransformation = m_chunkLayoutBuilder.transformationForTextBox(textBox);
-        if (textBoxTransformation.isIdentity())
-            continue;
+    auto boxesToTransform = textBoxes | std::views::filter([&](const auto& textBox) {
+        return !m_chunkLayoutBuilder.transformationForTextBox(textBox).isIdentity();
+    });
```

Comment:
I am not a big fan of this kind of changes because it is difficult to see their benefits. This statement adds an extra loop to filter the boxes. It also requires the extra memory for the filtered boxes `boxesToTransform`.